packages/web/src/components/gcds-link/stories/overview.mdx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Canvas, Story } from '@storybook/blocks';
+import * as Link from './gcds-link.stories';
+
+<Meta of={Link} name="Overview" />
+
+# Link<br/>`<gcds-link>`
+
+_Also called: call to action, CTA, link._
```

Comment:
This seems to still be using the button content. Can we change this to:
`Also called: anchor, hyperlink, call to action`